Don't search for libraries that are only used by the gtklos extension.
--- configure.orig	2023-03-11 07:47:23.000000000 -0600
+++ configure	2023-03-11 07:50:14.000000000 -0600
@@ -7249,6 +7249,7 @@
 HAVE_CANVAS=0
 GOOCANV=absent
 GOOVERS=""
+if false; then
 
 # Do we have GTK+?
 if ${PKGCONFIG} --exists gtk+-3.0
@@ -7276,6 +7277,7 @@
    GOOCANV=present
 fi
 
+fi
 ###
 ### Finishing ...
 ###
